.github/workflows/main.yml

```diff
@@ -0,0 +1,71 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Build pdf
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the main branch
+on:
+  push:
+    branches: [ main ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Runs a single command using the runners shell
+      - name: cd on workspace
+        run: cd $GITHUB_WORKSPACE
+
+
+      - name: Github Action for LaTeX
+        # You may pin to the exact commit or the version.
+        # uses: dante-ev/latex-action@259e5ff953ad4d4616fbd09cc87a526ee5c4499a
+        uses: xu-cheng/latex-action@v2
+        with:
+          # The root LaTeX file to be compiled
+          root_file: sample.tex
+          # The working directory for the latex compiler to be invoked
+          working_directory: . # optional
+          # LaTeX engine to be used
+          # compiler: pdflatex # optional, default is latexmk
+          # Extra arguments to be passed to the latex compiler
+          # args: ""
+          # Install extra packages by apt-get
+          # extra_system_packages: # optional
+
+                        
+      - name: Rename pdf
+        run: mv sample.pdf cv.pdf
```

Comment:
This can be set using the `-jobname=cv` option to `args` by compile latex

Reply:
Oo. Ok, that's nice ! (I'll not modify it since it works for my case, for my CV)
But cool!